01-2024-schedule/week-1.md

```diff
@@ -1,7 +1,112 @@
 ---
-description: Keys and Addresses, Wallet Recovery and Digital Signatures
+description: Building on bitcoin and setting up a bitcoin development environment
 ---
 
 # Week 1
 
-## Prerequisites
\ No newline at end of file
+## Prerequisites
+We would like you to download some source code and attempt to compile it by following its build instructions before the course begins. This is to prevent any delays when starting on the day.
+
+Please download/clone the following tools/repositories and try to follow and complete their build processes:
+
+| Software       | Repository                                                                                   |
+| -------------- | -------------------------------------------------------------------------------------------- |
+| TABconf wallet | [https://github.com/KayBeSee/tabconf-workshop](https://github.com/KayBeSee/tabconf-workshop) |
+| Bitcoin Core   | [https://github.com/bitcoin/bitcoin](https://github.com/bitcoin/bitcoin)                     |
+| LND            | [https://github.com/lightningnetwork/lnd](https://github.com/lightningnetwork/lnd)           |
+| Polar          | [https://lightningpolar.com/](https://lightningpolar.com)                                    |
+
+{% hint style="warning" %}
+Don't run `bitcoind` without specifying `regtest` (or `signet`) as the network in the configuration file (`bitcoin.conf`) or as a command line argument (see `bitcoind --help`). Otherwise you will start synchronising blocks for mainnet which is over 400 GB! At this point, you don't even need to run bitcoind just yet.
+{% endhint %}
+
+
+## Goals
+
+* Discuss and understand bitcoin philosophy
+* Become comfortable and proficient in setting up bitcoin developer environments
+* Be able to run and interface with Bitcoin Core and LND
+  * Understand signet and regtest
+  * Connect Bitcoin Core and LND programmatically
+  * Interface with Bitcoin Core and LND using the CLI tools
+  * Programmatic control of bitcoind and LND
+  * Comfortable with running the full test suite(s) of Bitcoin Core
+* Build a toy bitcoin wallet using a library
+  * Add additional functionality of your choosing
+* Develop a robust understanding of bitcoin topics both in theory and in practice:
+  * Random number generation
+  * Keys and key material
+  * Derivation paths
+  * Address types
+  * Coin selection
+  * The differences between keys, address and digital signatures
+* Gain experience with developing and using a Bitcoin wallet UI
+* Introduction to Technical Writing 
+  * Learn the critical basics of technical writing
+  * Develop skills for making documentation more accessible to all.
+
+
+## Monday & Tuesday
+
+1. Introduction to Qala
```

Comment:
I think you meant to write `Introduction to Btrust Builders Programme`.

Reply:
I have made the necessary changes